### PR TITLE
Scottish epc load command

### PR DESCRIPTION
**Pull request by LukaszDygon**
For now this is WIP, worth adding link to this to Swiki if approved, because imo it automates and simplifies at least the creation of data to be loaded into the DB

### CODE REVIEW

File: utils/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+### update_scottish_epc
+
+Currently WIP
+
+For now this creates a CSV file containing scottish EPC data ready to be loaded into the DB manually,
+as described in [Swiki article](https://softwiretech.atlassian.net/wiki/spaces/Support/pages/20520173599/Common+Tasks#How-to-actually-do-an-EPC-data-dump-for-fun-%26-profit).
+
+TODO: Add automatic loading to DB.
+
+The data is deduplicated (using latest only), sorted in order of files, with records without EPC number dropped.
```
Comment (JoshAdamPowell) [inline utils/README.md:23]: it's records without uprns that are dropped, UPRNs are property IDs (Unique Property Reference Number)

File: utils/README.md
Comment (JoshAdamPowell) [inline utils/README.md:1]: The process is near identical for English EPCs so we can probably modify it a little to work on either type :)

**Comment by CheshireSwift** [comment]
Will check more thoroughly next week, but at a glance this looks real nice, ty 🙏

**Comment by CheshireSwift** [comment]
Closing old PR (but retaining in GitHub for reference if required).

**Pull request closed without merging**